src/components/Search/SearchResults.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, FC } from "react";
+import useSongList from "../../hooks/useSongList";
+import { BrowserSessionContext } from "../../BrowserSessionProvider";
+import SelectInput from "ink-select-input/build";
+import { Text, useFocusManager, useInput } from "ink";
+import Gradient from 'ink-gradient';
+import Spinner from 'ink-spinner';
+
+type SRState = {
+  searchResultActive: boolean,
+  setSearchResultActive: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const SearchResults:FC<{state: SRState}> = ({state}) => {
+
+  const session = useContext(BrowserSessionContext);
+  const songList = useSongList();
+
+  const { focus } = useFocusManager();
+
+  useInput ((_, key) => {
+    if (!state.searchResultActive) return;
+
+    if (key.leftArrow) {
```

Comment:
intuitively I would expect to use esc here - maybe add `|| key.escape`?

Reply:
changing now. thank you! was wondering what should be the escape from results